com.github.makhber.Makhber.yaml

```diff
@@ -0,0 +1,63 @@
+app-id: com.github.makhber.Makhber
+runtime: org.kde.Platform
+sdk: org.kde.Sdk
+runtime-version: '5.15'
+command: makhber
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --filesystem=host
```

Comment:
Why filesystem host? This is very broad permission.

Reply:
The app write/read project files, should I restrict the user where to save his own files?
Any suggestion.

Reply:
These project files are cross-platform, the user can save a project on OS and load it on another.

Reply:
These project files are cross-platform, the user can save a project on OS and load it on another.

Reply:
`--filesystem=home` would be better

Reply:
The app need to save/load project files from external-media.

Reply:
you can still add `--filesystem=/media` and `--filesystem=/run/media`